### PR TITLE
[Feature] Add copy base uri button

### PR DESCRIPTION
**Pull request by BrettCleary**
This allows the user to use wallet connect with wallets that cannot parse out the base uri from the metamask deep link qr code

To test Ledger Live for example
1. Go to accounts
2. click on an account
3. click wallet connect icon in top right
4. paste URL you copied with the copy base uri button

#### Screenshots
wallet connect has base uri button
image

mm mobile does not have the base uri button
image

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Review by stormcakes — APPROVED** [review]
Worked perfectly for my Ledger :)

File: src/backend/utils.ts [hunk 1]
```diff
@@ -86,7 +85,6 @@ import {
   updateWineVersionInfos,
   wineDownloaderInfoStore
 } from './wine/manager/utils'
-import { clean } from 'easydl/dist/utils'
```
Comment (flavioislima) [inline src/backend/utils.ts:89]: Is this change related? also the translations update? or it came from another branch?

**Review by flavioislima — APPROVED** [review]
I think looks fine, but the branch has some unrelated changes I guess. 👍🏽

File: src/backend/utils.ts (hunk 1 above)
Comment (BrettCleary) [reply src/backend/utils.ts:89]: These variables were unused. Not sure how they got back into develop after your replaced unzipper with the other library.
